testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -183,6 +183,11 @@ public void testViewReferencingTableInDifferentSchema()
         onHive().executeQuery(format("CREATE VIEW %s.%s AS SELECT * FROM %s.%s", schemaX, viewName, schemaY, tableName));
 
         assertThat(onTrino().executeQuery(format("SELECT COUNT(*) FROM %s.%s", schemaX, viewName))).containsOnly(row(25));
+
+        onHive().executeQuery(format("DROP TABLE %s.%s", schemaY, tableName));
```

Comment:
You seem to have switched in the meantime from randomSuffix table names to the current format.
I thought that using the randomSuffix was intentional to  avoid therefore to execute further `DROP` SQL statements on Hive.

What was the reason behind this change?

Reply:
Schema has random suffix so everything inside it doesn't need randomness. Names are shorter and easier to read/follow.